app/src/main/java/de/westnordost/streetcomplete/quests/tactile_paving/AddTactilePavingBusStop.kt

```diff
@@ -31,12 +31,11 @@ class AddTactilePavingBusStop : OsmFilterQuestType<Boolean>() {
     override fun getTitle(tags: Map<String, String>): Int {
         val hasName = tags.containsKey("name")
         val isTram = tags["tram"] == "yes"
-        return if (isTram) {
-            if (hasName) R.string.quest_tactilePaving_title_name_tram
-            else         R.string.quest_tactilePaving_title_tram
-        } else {
-            if (hasName) R.string.quest_tactilePaving_title_name_bus
-            else         R.string.quest_tactilePaving_title_bus
+        return when {
+            isTram && hasName ->    R.string.quest_tactilePaving_title_name_tram
+            isTram ->               R.string.quest_tactilePaving_title_tram
+            hasName ->              R.string.quest_tactilePaving_title_name_bus
+            else ->                 R.string.quest_tactilePaving_title_bus
```

Comment:
Matter of taste I guess (the other as well). I slightly prefer the current version but I am fine with your version too